seed/seed.go

```diff
@@ -73,14 +79,29 @@ func (s *Seeder) Seed(cfg *config.Config) error {
 			}
 			value, err := gtr.Get(key)
 			if err != nil {
-				return err
+				log.Errorf("failed to get consul key %s for field %s: %v", key, f.Name, err)
+				continue
+			}
+			if value == nil {
+				log.Warnf("consul key %s did not exist for field %s", key, f.Name)
+				continue
 			}
```

Comment:
That block could be omitted if [here](https://github.com/taxibeat/harvester/blob/88c7414d745a578ca198d658cca737594c55e968/seed/consul/getter.go#L50) instead of `nil, nil` you return an error like `nil, fmt.Errorf("consul key %s did not exist", key)`

Reply:
We log differently in each case. the getter get a log.Error and the missing key is a log.Warn.